tests/foreman/api/test_location.py

```diff
@@ -66,6 +64,19 @@ def _make_proxy(self, options=None):
         self.addCleanup(capsule_cleanup, proxy['id'])
         return proxy
 
+    @classmethod
+    def setUpClass(cls):
```

Comment:
From what I see, we usually also do something like:
```
super(LocationTestCase, cls).setUpClass()
```

Which will call `setUpClass` method of `APITestCase`.

Reply:
great chatch, updated